### PR TITLE
EPMLABSBRN-244 githooks were added to the repository

### PR DESCRIPTION
**Pull request by goopnigoop**
[EPMLABSBRN-244](https://jira.epam.com/jira/browse/EPMLABSBRN-244)

githooks were added to the project:

- [x] **pre-commit** (commit message should have prefix: EPMLABSBRN-X)
- [x] **pre-push** (no push to dev/master)
- [x] **commit-msg** (no commit to dev/master)

- a command to set core.hooksPath was added to Makefile
- core.hooksPath automatically set during the build

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+master_branches=('origin/master' 'master' 'origin/develop' 'develop')
```
Comment (ElenaSpb) [inline .githooks/pre-commit]: our current dev branch is
**origin/dev**

File: .githooks/pre-push [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+protected_branch_master='master'
+protected_branch_dev='dev'
```
Comment (ElenaSpb) [inline .githooks/pre-push]: is `dev` ok?
or it should be `origin/dev`?

File: .githooks/pre-commit (hunk 1 above)
Comment (goopnigoop) [reply .githooks/pre-commit]: thanks, I'll fix it

File: .githooks/pre-push (hunk 2 above)
Comment (goopnigoop) [reply .githooks/pre-push]: it's ok

**Pull request merged**